src/library_syscall.js

```diff
@@ -1310,8 +1315,13 @@ var SyscallsLibrary = {
 #if ASSERTIONS
     assert(!flags);
 #endif
-    if (old.fd === suggestFD) return -{{{ cDefine('EINVAL') }}};
-    return SYSCALLS.doDup(old.path, old.flags, suggestFD);
+    if (old.fd === suggestFD) return -ERRNO_CODES.EINVAL;
```

Comment:
switch these to use the cDefine to avoid a dep on ERRNO_CODES which, I believe was removed